code/android-lifecycle-library/build.gradle

```diff
@@ -0,0 +1,66 @@
+apply plugin: 'com.android.library'
+apply plugin: "jacoco"
+
+android {
+    compileSdkVersion 30
+
+    defaultConfig {
+        minSdkVersion 19
+        //noinspection OldTargetApi
+        targetSdkVersion 30
+        //Include the Proguard rules for Core Extension in the aar
+        consumerProguardFiles 'lib-proguard-rules.pro'
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+    }
+
+    sourceSets {
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+        unitTests.includeAndroidResources = true
+    }
+
+    flavorDimensions "target"
+
+    productFlavors {
+        phone {
+            dimension "target"
+        }
+    }
+
+    buildTypes {
+        debug {
+            testCoverageEnabled false
+            debuggable true
+        }
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation project(path: ':android-core-library')
+    // unit tests
+    testImplementation "junit:junit:4.13.2"
+    testImplementation project(path: ':android-lifecycle-library')
```

Comment:
Now we can remove this "testImplementation", it was a workaround for fixing a bug of the latest Android Studio.